index.js

```diff
@@ -29,20 +33,104 @@ cp.parseBlock = function () {
 	return this.finishNode(node, "BlockStatement");
 };
 
+function strToModuleIdentifier(str) {
```

Comment:
That would be rad but not supported in Node 4. I should keep tabs on all the bummers Node 4 support causes :D